lib/grape/dsl/inside_route.rb

```diff
@@ -191,8 +191,12 @@ def body(value = nil)
       #
       #   GET /file # => "contents of file"
       def file(value = nil)
-        if value
-          @file = Grape::Util::FileResponse.new(value)
+        if value.is_a?(String)
+          file_body = Grape::ServeFile::FileBody.new(value)
+          @file = Grape::ServeFile::FileResponse.new(file_body)
+        elsif !value.is_a?(NilClass)
+          warn '[DEPRECATION] Argument as FileStreamer-like object will be deprecated. Use path to file instead'
```

Comment:
I mean that support of argument as FileSteam-like object will be removed from Grape in next version after 0.15.1 or 0.16.0. After releasing 0.15.1 or 0.16.0 I would like to remove it at all. Sorry for my English :)
OK, I'll replace it with:

```
[DEPRECATION] Argument as FileStreamer-like object is deprecated. Use path to file instead.
```